src/components/Dashboard.js

```diff
@@ -48,23 +62,22 @@ class Dashboard extends Component {
         <Menu />
         <h1>Dashboard</h1>
         <a href={this.state.contact + this.state.dangerMessage}>
-          <Button color='danger' onClick={this.toggle}>
+          <Button color='danger' onClick={this.handleClick}>
             <h3>I need urgent help</h3>
             <p>TBC</p>
           </Button>
         </a>
         <br />
         <br />
         <a href={this.state.contact + this.state.vulnerableMessage}>
-          <Button color='warning' onClick={this.toggle}>
+          <Button color='warning' onClick={this.handleClick}>
             <h3>I&#39;m feeling vulnerable</h3>
           </Button>
         </a>
         <SafeModal toggle={this.toggle} modal={this.state.modal} />
-        {console.log(this.state.userLocation)}
         <br />
         <br />
-        <a href={`http://www.google.com/maps/place/${this.state.userLocation.latitude},${this.state.userLocation.longitude}`}>See Where I am</a>
+        {this.state.showUserLocation ? <a href={`http://www.google.com/maps/place/${this.state.userLocation.latitude},${this.state.userLocation.longitude}`}>See Where I am</a> : null}
```

Comment:
you can do {this.state.showUserLocation && ... } if there is only one condition, then you don't need the ': null'